app/assets/stylesheets/_variables.scss

```diff
@@ -1,52 +0,0 @@
-// COLORS
```

Comment:
Why aren't we using a dedicated variables file any more?

Reply:
I re-added the variables file as discussed